docs/ko/docs/tutorial/middleware.md

```diff
@@ -0,0 +1,61 @@
+# 미들웨어
+
+당신은 미들웨어를 **FastAPI** 어플리케이션에 추가할 수 있습니다.
+
+"미들웨어"는 특정 *path operation*에 의해 처리되기 전, 모든 **request**에 대해서 동작하는 함수입니다. 또한 모든 **response**가 반환되기 전에도 동일하게 동작합니다.
+
+* 미들웨어는 당신의 어플리케이션으로 오는 **request**를 가져옵니다.
+* **request** 또는 다른 필요한 코드를 실행 시키는 동작을 할 수 있습니다.
+* **reqeust**를 전달하여 일부 어플리케이션에서 처리합니다 (일부 *path 작업 *에 의해).
+* 어플리케이션에서 생성한 **response**를 받습니다 (일부 *path 작업 *에 의해).
+* **response** 또는 다른 필요한 코드를 실행시키는 동작을 할 수 있습니다.
+* **response**를 반환합니다. 그런 다음 ** 요청 **을 전달하여 나머지 응용 프로그램에서 처리합니다 (일부 *path 작업 *에 의해).
+
+!!! note "기술 세부사항"
+    만약 당신이`yield`를 사용한 의존성을 가지고 있다면, 미들웨어가 실행되고 난 후에 exit이 실행될 것입니다.
+
+    만약 백그라운드 작업이 있다면, 그것들은 모든 미들웨어가 실행되고 나서 실행될 것입니다.
```

Comment:
```suggestion
    만약 (나중에 문서에서 다룰) 백그라운드 작업이 있다면, 모든 미들웨어가 실행되고 *난 후에* 실행됩니다.
```

The author says that there's a document about background tasks via description with "(documented later)". Therefore, you must not get rid of it.